6_object_wrap/node-addon-api/package.json

```diff
@@ -5,8 +5,11 @@
   "main": "addon.js",
   "private": true,
   "gypfile": true,
+  "engines": {
+    "node": ">= 14.0.0"
+  },
```

Comment:
I don't think we'd want to add this. If the example needs to be different for multi-context and that is only available in later Node.js versions, then we should add a new example versus changing the old one. Ideally we would have an example that would work across all Node.js LTS versions.

Reply:
Ok, I changed it and it a new example now. But there is linked bug  https://github.com/nodejs/node-addon-api/issues/730